.github/actions/install-dashboards/action.yml

```diff
@@ -39,8 +39,8 @@ runs:
 
     - id: osd-version
       run: |
-        echo "::set-output name=osd-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-4)"
-        echo "::set-output name=osd-x-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-3)"
+        echo "::set-output name=osd-version::$(jq -r '.opensearchDashboards.version' package.json)"
```

Comment:
Here is a little break down of the first command:
- -r (or --raw-output): This option tells jq to output raw strings instead of JSON-encoded strings, which means it won't wrap the output in quotes.

Reply:
If `.opensearchDashboards.version` is `2.10.0` will this produce the following?

```
osd-version = 2.10
osd-x-version = 2.x
```

I believe that's what the current logic intends.

Reply:
Yeah, for the x-version, I moved the `.x` into this step. so it will be "2.x" directly.

Reply:
`jq -r '.opensearchDashboards.version' package.json` will output 2.10.0 right? This variable is used to checkout a branch [here](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/.github/actions/install-dashboards/action.yml#L51) so it should match the branch name `2.10`.

Reply:
@cwperks nice catch! My bad, I wasnt noticing that at the beginning. I just fixed with:
```
echo "::set-output name=osd-version::$(jq -r '.opensearchDashboards.version | split(".") | .[:2] | join(".")' package.json)"
```
This will be the same strategy as the `[0]` we used down below. But `[0]` is just getting the first element, and this line will get the first 2 element of the array by using `[:2]`.